EventSubscriber/CompleteConsoleCommandEventSubscriber.php

```diff
@@ -70,19 +69,17 @@ public function onConsoleTerminate(ConsoleTerminateEvent $event)
             return;
         }
 
-        try {
-            // lookup job log and save status
-            $log = $this->jobLogRepository->getJobLog($uuid);
-
-            if($log->getStatus() === JobLog::STATUS_RUNNING) {
-                $log->setStatus(JobLog::STATUS_COMPLETE);
-            }
-            $log->setPeakMemoryUse($this->getPeakMemoryUse());
-            $log->setCompleted((new \DateTime('now'))->format('U'));
-            $this->jobLogRepository->save($log);
-        } catch (UnknownJobLogException $e){
-            // forgive exceptions
+        // lookup job log and save status
+        $log = $this->jobLogRepository->findJobLog($uuid);
+        if (!$log) {
+            // if no log, ignore it
```

Comment:
Missing return statement

Reply:
Thanks 👍 Updated

Reply:
Seems like we might want to introduce phpstan here, seems to be a bunch of things like this gone unnoticed.